web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -76,20 +78,37 @@ function joinup_core_requirements($phase): array {
     $requirements['joinup_core_forward_revisions'] = [
       'title' => t('Forward published revisions'),
       'description' => t('In all content, the last published revision should be the latest of the published revisions.'),
+      'severity' => REQUIREMENT_OK,
+      'value' => t('No entities were found without a parent.'),
     ];
 
-    /** @var \Drupal\joinup_core\RequirementsHelper $requirements_helper */
-    $requirements_helper = \Drupal::service('joinup_core.requirements_helper');
     $results = $requirements_helper->getNodesWithProblematicRevisions();
     if (!empty($results)) {
       $nids = array_keys($results);
       $error = t('Issues with the latest revision have been found in the following nid(s): :nids', [
         ':nids' => implode(', ', $nids),
       ]);
-      $requirements['joinup_core_forward_revisions'] += [
-        'severity' => REQUIREMENT_ERROR,
-        'value' => $error,
-      ];
+      $requirements['joinup_core_forward_revisions']['severity'] = REQUIREMENT_ERROR;
+      $requirements['joinup_core_forward_revisions']['value'] = $error;
+    }
+
+    // Regression requirement to ensure that no orphaned triples are leftover
+    // after various actions that interact with SPARQL storage.
+    $requirements['joinup_core_orphaned_triples'] = [
```

Comment:
As we agreed, we'll have only one status report requirement, in `joinup_rdf` module, called “RDF content integrity” (or a better name, if you find). Each type of data integrity violation will be added to this requirement, as a bullet or other categorisation markup. The logic encapsulated in the helper should be also moved in there.